imageprocessor/tests.py

```diff
@@ -41,6 +41,29 @@ def test_register_page(self):
         response = client.get(reverse('register'))
         self.assertTrue(response.status_code == 200)
 
+    def test_not_logged_in_user_cannot_view_my_pictures(self):
+        client = Client()
+        response = client.get(reverse('view_my_pictures'))
+        self.assertTrue(response.status_code == 302)
+
+    def test_logged_in_user_can_view_my_pictures(self):
+        client = Client()
+        client.post(reverse('register'),{'username': "TestUser1", 'password1': "testpassword1", 'password2': "testpassword1"})
+        client.login(username="TestUser1", password="testpassword1")
+        response = client.get(reverse('view_my_pictures'))
+        self.assertTrue(response.status_code == 200)
+
+    def test_view_my_pictures_picture_count(self):
+        client = Client()
+        response = client.post(reverse('register'),{'username': "TestUser1", 'password1': "testpassword1", 'password2': "testpassword1"})
+        client.login(username="TestUser1", password="testpassword1")
+        images_to_upload = 5
+        for i in range(images_to_upload):
+            with open(TEST_IMAGES_DIR + "/image1.jpg", "rb") as file:
+                    client.post(reverse('classify'), {'file': file})
```

Comment:
You should actually be able to verify that the correct url to the image and the correct tags appear in the view_my_pictures page

Reply:
Could you also make sure the image is being deleted from cloudinary after the test runs? There should be a couple examples of this elsewhere in this file, it just keeps us from having to go in and delete them or have a bunch of duplicates